### DISCUSSION TITLE
Option to manually enter ports that each machine is using

### DISCUSSION THREAD

**Discussion by Krajcara** [gameserverapp/Platform#1016] [Suggestions]
On a settings page of the machine add the options to enter ports that machine will use. 
For FTP, SSH and Passive FTP ports that option is not necessary only for game ports.
Game port: from - to
Query port: from - to
RCON port: from - to
Atlas Seamless port: from - to
Atlas map port: from - to
Reason is that we need to open much more ports then we ever use per one machine. I think no one will host 200+ games on one machine.

**Comment by mvaessen** [comment]
Thanks for the suggestion, but I'm unfortunately going to reject this. Current setup works great and I see no benefits for this
